### ISSUE TITLE
add Telegram

### ISSUE THREAD

**Issue by stevenwagner** [nylira/prism-break#1075]
Open source instant messaging with encryption. Telegram.org

**Comment by azzamsa** [comment]
i come here also want to know why "Telegram" is not in the list.
i have been using this app for 2 years, and recommend it to othes. sadly, i never research it my self.

Thank you so much for pointing out this great article.
http://www.cryptofails.com/post/70546720222/telegrams-cryptanalysis-contest

thanks for explaining @nylira

**Comment by sedrubal** [comment]
Telegram Android source code is IMHO not FOSS (see https://github.com/DrKLO/Telegram/pull/76). I don't want to recommend it.

**Comment by Felicis** [comment]
**TL;DR: Why not mention Telegram in the Notes, saying its server side is not FLOSS instead of not mentioning it at all?**

I have seen Telegram is not in the list (as of november 2017).  
I share the opinion that it is unsuited for being recommended here on prism-break as their server software is not FLOSS. Also the article http://www.cryptofails.com/post/70546720222/telegrams-cryptanalysis-contest raises doubts about their crypto-system (although the article is now almost 4 years old, and Telegram has changed since, so not all claims are still true).  
Nevertheless I consider Telegram a strong player among the privacy protecting messengers, and that's why I asked myself in the first place why it's not in the list (I admit I didn't know about the proprietary server thing, though...). Therefore I suggest to simply mention Telegram in the Notes section and explain the reasons for not taking it into the recommendation list, i.e. that the server side is not FLOSS and there is some doubt about its crypto (linking the cryptofails-article). Just as with Threema, which is a good privacy-protecting IM, just not FLOSS.  

Sidenote: I don't still don't really get the idea behind the proprietary list next to the recommended one... Proprietary doesn't sound evil enough (to me) and it could be misunderstood, that the proprietary list were the next best choice after the recommended list, which I'm sure isn't intended!

PS: Almost all my contacts are using mainly WhatsApp, but due to the ease of use and similarity to WhatsApp, many of them also use Telegram. As very few of my friends would follow me to one of the messengers listed here :sob: I believe Telegram to be more suitable for the masses. Therefore I use Telegram for communication with everyone I can't get to the really secure IMs. I believe lots of people (I know) still aren't ready for the really secure solutions, while interested in gradually increasing their privacy, so it could be worth compiling a list of software which takes the normal user (e.g. my grandma) away from google & co (which would also raise the sensibility for privacy with common folks). I have found no section on this website mentioning the intended audience or scope of the presented software, so I don't know if non-hardcore solutions are meant to figure.

**Comment by strugee** [comment]
Probably the right solution here is #1686
